server/datastore/mysql/microsoft_mdm.go

```diff
@@ -371,7 +371,7 @@ func updateMDMWindowsHostProfileStatusFromResponseDB(
 	// update their detail and status.
 	const updateHostProfilesStmt = `
 		INSERT INTO host_mdm_windows_profiles
-			(host_uuid, profile_uuid, detail, status)
+			(host_uuid, profile_uuid, detail, status, command_uuid)
```

Comment:
not sure if I'm tired, but can't understand why the integration test for this is wasn't failing (`TestIntegrationsMDM/TestWindowsProfileManagement`). Without this, I get the following error:

```
Error 1364 (HY000): Field 'command_uuid' doesn't have a default value
```

I have verified that the tests indeed update the `status` value in the database, so I'm at loss. Maybe we're setting a magic MySQL var for tests? I would like to see a failing test get fixed by this to be comfortable

Reply:
My guess would be that the test exercises the ON DUPLICATE KEY UPDATE part of the statement, and mysql detects the duplicate key state before applying the validations of the insert? In any case, that sounds like something I will need in the feature branch of OS updates, right (i.e. that would result in failures deploying profiles to hosts?)